src/index.js

```diff
@@ -0,0 +1,327 @@
+import keyboardElements from './keyboardObject.js';
+import {createKeyboard, mainDivId} from "./createKeyboard.js";
+import {createTextArea, textAreaId, changeTextAreaCursorPosition, insertValueIntoTextArea, getTextAreaCommonCursorPosition,
+    calculateTextAreaCursorPositionAfterDown, calculateTextAreaCursorPositionAfterUp, createCommentText} from "./textAreaObject.js";
+
+
+const keyboardLanguages = {eng: 'eng', ru: 'ru'};
+let shiftBtn = false;
+let altBtn = false;
+let capsBtn = false;
+let currLang = keyboardLanguages.eng;
+
+createTextArea();
+createKeyboard(keyboardElements);
+
+const textarea = document.querySelector(textAreaId);
+const mainDiv = document.getElementById(mainDivId);
+
+createCommentText(mainDiv);
+
+function getCorrespondentCaseKeyboardValue(){
+    if (shiftBtn){
+        return 'caseUp';
+    }
+    else if (capsBtn){
+        return 'caps';
+    }
+    return 'default';
+}
+
+function globalPressActionHandler(event, keycode, target){
+    let resultValue;
+    target.classList.add('selected-button');
+    try {
+        resultValue = keyboardElements.find(key => key.keycode === keycode).values[currLang][getCorrespondentCaseKeyboardValue()];
+    }
+    catch (e) {}
```

Comment:
It's not async action, so it's wrong to use try catch here (but it works as expected here). You want to find and check 'values' and if an object has it, set currLang.
So instead of using try catch here, just go with simple 'if'